### ISSUE TITLE
Crash on upgrade to v1.19.0 on old androids due to recursive deletion utility

### ISSUE THREAD

**Issue by imsodin** [syncthing/syncthing-android#1753]
There's a crash related to the recursive deletion utility function used, occurring on old (I think <=7) androids only:

```
java.lang.NoSuchMethodError: 
  at org.apache.commons.io.FileUtils.forceDelete (FileUtils.java:1341)
  at org.apache.commons.io.FileUtils.cleanDirectory (FileUtils.java:324)
  at org.apache.commons.io.FileUtils.deleteQuietly (FileUtils.java:1236)
  at com.nutomic.syncthingandroid.activities.FirstStartActivity.upgradeToApiLevel30 (FirstStartActivity.java:213)
  at com.nutomic.syncthingandroid.activities.FirstStartActivity.access$500 (FirstStartActivity.java:51)
  at com.nutomic.syncthingandroid.activities.FirstStartActivity$ViewPagerAdapter$3.onClick (FirstStartActivity.java:333)
  at android.view.View.performClick (View.java:6312)
  at android.widget.TextView.performClick (TextView.java:11202)
  at android.view.View$PerformClick.run (View.java:23985)
  at android.os.Handler.handleCallback (Handler.java:751)
  at android.os.Handler.dispatchMessage (Handler.java:95)
  at android.os.Looper.loop (Looper.java:154)
  at android.app.ActivityThread.main (ActivityThread.java:6816)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run (ZygoteInit.java:1563)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1451)
```

We should do the deletion manually instead.

**Comment by Catfriend1** [comment]
Hi, some quick win here would be to use our shell wrapper "runShellCommand" and put a "rm -rf /syncthingdir/db/" in there. This works Independent from API.

**Comment by Catfriend1** [comment]
👍

**Comment by progserega** [comment]
Also crash on update 1.19.0.2
I hope fix from previous comment will help, but I attach log from 1.19.0.2 - may be it will help otherwise.
[logcat_fail_1.19.0.2.log](https://github.com/syncthing/syncthing-android/files/8037009/logcat_fail_1.19.0.2.log)

**Comment by imsodin** [comment]
This is finally really fixed in 1.19.0.3, which is released since yesterday.
